test/views/components/LateSubmissionPenaltySummaryCardSpec.scala

```diff
@@ -637,7 +637,7 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
       }
 
       "shows the financial heading WITHOUT point number when the point is above threshold for filing frequency and a rewording of the appeal text" in {
-        docWithFinancialLSP.select(".app-summary-card__title").get(0).text() shouldBe "£200 penalty"
+        docWithFinancialLSP.select(".app-summary-card__title").get(0).text() shouldBe "£200 penalty for late submission of VAT due on 30 October 2069"
```

Comment:
May be worth having a check for the visible text and the hidden text separately:
```suggestion
        docWithFinancialLSP.select(".app-summary-card__title").get(0).ownText() shouldBe "£200 penalty"
        docWithFinancialLSP.select(".app-summary-card__title span").get(0).ownText() shouldBe "for late submission of VAT due on 30 October 2069"
```